src/battle_ai_main.c

```diff
@@ -2610,6 +2611,13 @@ static s16 AI_CheckBadMove(u8 battlerAtk, u8 battlerDef, u16 move, s16 score)
             if (gBattleMons[battlerAtk].hp <= gBattleMons[battlerAtk].maxHP / 3)
                 score -= 10;
             break;*/
+        case EFFECT_TAKE_HEART:
+            if ((!(gBattleMons[battlerAtk].status1 & STATUS1_ANY)
+             || PartnerHasSameMoveEffectWithoutTarget(BATTLE_PARTNER(battlerAtk), move, AI_DATA->partnerMove))
```

Comment:
No? This is a single-target move. It has no bearing on what the partner does.

Reply:
I was asked to add it.

Reply:
I can't see where in this thread, but if anything, it should check if the partner is already using Jungle Healing or Heal Bell/Aromatherapy.

Reply:
It was inside the Discord server. We make comments and sometimes discuss stuff concerning the PRs over there.

Reply:
Feel free to join if you want :)

https://discord.gg/6CzjAG6GZk